File: Assignment5/program4.c
#include<stdio.h>
// TIME COMPLEXITY = O(N)  {O(N/2) WHICH IS O(N)}
void OddDisplay(int iNo)
{
  int iCnt=0;
  for(iCnt=1;iCnt<=iNo;iCnt+=2)
  {
    if((iCnt % 2) != 0)
    printf("%d ",iCnt);
  }
  
}

int main()
{
  int iValue = 0;
  printf("Enter the number : ");
  scanf("%d",&iValue);
  OddDisplay(iValue);
  return 0;
}

File: Assignment11/program4.java
import java.util.Scanner;
// TIME COMPLEXITY O(N)
class Pattern
{
  public void Display(int iCol)
  {
    int i = 0;
    for(i = 1;i<=iCol;i++)
    {
      System.out.print("#\t"+i+"\t*\t");
    }
  }
}
public class program4
{
  public static void main(String[] args)
   {
      int iValue = 0;
      Scanner sobj = new Scanner(System.in);
      System.out.println("Enter the number of Elements: ");
      iValue = sobj.nextInt();
      Pattern pobj = new Pattern();
      pobj.Display(iValue);

  }
}
